src/main/java/org/fundacion/jala/converter/view/login/LoginInterface.java

```diff
@@ -32,93 +44,125 @@
+        CLOSE_BTN.setFont(new Font("Barlow", Font.BOLD, FONT_SIZE));
+        CLOSE_BTN.setOpaque(true);
+        CLOSE_BTN.setFocusPainted(false);
+        CLOSE_BTN.setBackground(null);
+        CLOSE_BTN.setForeground(Color.WHITE);
+        Border border = new LineBorder(Color.WHITE, 0);
+        CLOSE_BTN.setBorder(border);
+        SHOW_PASSWORD_BUTTON.setPreferredSize(new Dimension(BUTTON_EYE_SIZE, BUTTON_EYE_SIZE));
+        SHOW_PASSWORD_BUTTON.setOpaque(true);
+        SHOW_PASSWORD_BUTTON.setFocusPainted(false);
+        SHOW_PASSWORD_BUTTON.setBackground(null);
+        SHOW_PASSWORD_BUTTON.setBorder(border);
     }
 
     /**
-     * Adds the actions performed.
+     * Sets all the Panels and positions for components.
      */
-    public void addActionEvent() {
-        LOGIN_BUTTON.addActionListener(this);
-        SHOW_PASSWORD_BUTTON.addActionListener(this);
-        REGISTER_BUTTON.addActionListener(this);
+    public void setPanels() {
+        userPanel = new JPanel();
+        userPanel.setLayout(new FlowLayout(FlowLayout.CENTER));
+        userPanel.setBorder(new EmptyBorder(0, MARGIN_SPACE, 0, MARGIN_SPACE));
+        userPanel.setLayout(new FlowLayout(FlowLayout.LEFT, HORIZONTAL_SPACE_FLOW, VERTICAL_SPACE_FLOW));
+        userPanel.add(USERNAME_LABEL);
+        userPanel.add(USERNAME_TEXT_FIELD);
+        userPanel.setBackground(Color.DARK_GRAY);
+        passPanel = new JPanel();
+        passPanel.setBorder(new EmptyBorder(0, MARGIN_SPACE, 0, MARGIN_SPACE));
+        passPanel.setLayout(new FlowLayout(FlowLayout.LEFT, HORIZONTAL_SPACE_FLOW, VERTICAL_SPACE_FLOW));
+        passPanel.setBackground(Color.DARK_GRAY);
+        passPanel.add(PASSWORD_LABEL);
+        passPanel.add(PASSWORD_FIELD);
+        passPanel.add(SHOW_PASSWORD_BUTTON);
+        btnPanel = new JPanel();
+        btnPanel.setLayout(new FlowLayout(FlowLayout.CENTER, HORIZONTAL_SPACE_FLOW, VERTICAL_SPACE_FLOW));
+        btnPanel.add(LOGIN_BUTTON);
+        btnPanel.add(REGISTER_BUTTON);
+        btnPanel.setBackground(Color.DARK_GRAY);
+        JPanel iconContainer = new JPanel();
+        iconContainer.setLayout(new FlowLayout(FlowLayout.CENTER));
+        iconContainer.setBorder(new EmptyBorder(HORIZONTAL_SPACE_FLOW, 0, HORIZONTAL_SPACE_FLOW, 0));
+        iconContainer.add(loginImg);
+        iconContainer.setBackground(Color.DARK_GRAY);
+        mainContainer = new JPanel();
+        mainContainer.setLayout(new BoxLayout(mainContainer, BoxLayout.Y_AXIS));
+        mainContainer.add(iconContainer);
+        mainContainer.add(userPanel);
+        mainContainer.add(passPanel);
+        mainContainer.setBackground(Color.DARK_GRAY);
+        container = new JPanel();
+        container.setLayout(new FlowLayout(FlowLayout.RIGHT));
+        container.setBackground(Color.DARK_GRAY);
+        CLOSE_BTN = new JButton(new AbstractAction("X") {
+            @Override
+            public void actionPerformed(ActionEvent e) {
```

Comment:
The variable should be final and have a more descriptive name.

Reply:
Solved!